lib/src/base-styles/_nx-tile.scss

```diff
@@ -157,8 +157,21 @@ $nx-tile-background-color: $nx-white;
 
   border-top: 1px solid #e9e9e9;
   margin-top: $nx-spacing-md;
-  padding: $nx-spacing-xl 0;
+  padding: $nx-spacing-l 0;
   text-align: right;
+
+  .nx-btn-bar {
+    margin: 0;
+    text-align: right;
+  }
+
+  .nx-alert {
```

Comment:
Where can I see this scenario demonstrated?

Reply:
> Where can I see this scenario demonstrated?

Form layout page

Reply:
> Form layout page

That's a form footer not a tile footer

Reply:
> > Form layout page
> 
> That's a form footer not a tile footer

I added an example here: https://github.com/sonatype/sonatype-react-shared-components/pull/124/commits/1c2eaf488ea5a9cca068f88e984929d2f8c4649d